django/db/models/fields/related.py

```diff
@@ -1954,6 +1997,11 @@ class OneToOneField(ForeignKey):
     related_accessor_class = SingleRelatedObjectDescriptor
     description = _("One-to-one relationship")
 
+    many_to_many = False
```

Comment:
should these be listed above the other attributes of the class for consistency with the other fields? Also missing `# Field flags` comment like the others.